lib/sqlalchemy/dialects/sqlite/base.py

```diff
@@ -908,6 +909,8 @@ def get_column_specification(self, column, **kwargs):
         colspec = self.preparer.format_column(column) + " " + coltype
         default = self.get_column_default_string(column)
         if default is not None:
+            if isinstance(column.server_default.arg, functions.Function):
```

Comment:
**mike bayer** (zzzeek) wrote:

shouldn't we be rendering parenthesis around *any* expression?  am adding tests now